ui/components/multichain/connected-site-menu/connected-site-menu.test.js

```diff
@@ -13,17 +14,36 @@ import {
 import { ConnectedSiteMenu } from '.';
 
 describe('Connected Site Menu', () => {
-  const selectedAddress = '0xec1adf982415d2ef5ec55899b9bfb8bc0f29251b';
-
-  const identities = {
-    '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc': {
-      address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
-      name: 'Account 1',
-    },
-    '0xec1adf982415d2ef5ec55899b9bfb8bc0f29251b': {
-      address: '0xec1adf982415d2ef5ec55899b9bfb8bc0f29251b',
-      name: 'Account 2',
+  const internalAccounts = {
+    accounts: {
+      'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3': {
+        address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+        id: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
+        metadata: {
+          name: 'Account 1',
+          keyring: {
+            type: 'HD Key Tree',
+          },
+        },
+        options: {},
+        methods: [...Object.values(EthMethod)],
+        type: EthAccountType.Eoa,
+      },
+      '07c2cfec-36c9-46c4-8115-3836d3ac9047': {
+        address: '0xec1adf982415d2ef5ec55899b9bfb8bc0f29251b',
+        id: '07c2cfec-36c9-46c4-8115-3836d3ac9047',
+        metadata: {
+          name: 'Account 2',
+          keyring: {
+            type: 'HD Key Tree',
+          },
+        },
+        options: {},
+        methods: [...Object.values(EthMethod)],
```

Comment:
This isn't future proof since new methods will be added (signing UserOps, for example)